db/projects/kyber-network/asset-knc-logo-square.svg

```diff
@@ -0,0 +1 @@
+<svg id="svg2" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 461.9 801.9" width="1440" height="2500"><style>.st0{fill:url(#rect4147_1_)}.st1{fill:#e5f4f1}.st2{fill:#a7d8d2}.st3{fill:url(#path4155_1_)}.st4{stroke:#fff;stroke-width:2.5222;stroke-linejoin:bevel}</style><g id="layer1"><g id="g4228" transform="translate(660.895 -400.733) scale(3.50002)"><linearGradient id="rect4147_1_" gradientUnits="userSpaceOnUse" x1="-313.952" y1="305.091" x2="-316.095" y2="70.091" gradientTransform="matrix(2.8 0 0 -2.8 721.756 980.573)"><stop offset="0" stop-color="#54d29f"/><stop offset="1" stop-color="#249eac"/></linearGradient><path id="rect4147" class="st0" d="M-182.6 172.8l46.2-51 2.1 214.4-36.7-51.4-11.6-112z"/><path id="path4150" class="st1" d="M-131.1 205l-1-84.3 69.4 50.8-68.4 33.5z"/><path id="path4153" class="st2" d="M-63.2 280.8l-66.1-71.1 67.6-33.9-1.5 105z"/><linearGradient id="path4155_1_" gradientUnits="userSpaceOnUse" x1="-292.052" y1="232.417" x2="-299.195" y2="371.703" gradientTransform="matrix(2.8 0 0 -2.8 721.756 980.573)"><stop offset="0" stop-color="#008597"/><stop offset="1" stop-color="#00aa8f"/></linearGradient><path id="path4155" class="st3" d="M-128.6 340l-2.4-125 66 71.2-63.6 53.8z"/><circle id="path4157-3" class="st4" cx="-184.3" cy="173.3" r="4.2"/><circle id="path4157-6" class="st4" cx="-135.7" cy="119" r="4.2"/><ellipse id="path4157-7" transform="matrix(.9999 -.0122 .0122 .9999 -2.125 -.736)" class="st4" cx="-61.4" cy="173.9" rx="14.7" ry="14.7"/><circle id="path4157-5" class="st4" cx="-134.6" cy="209.9" r="4.2"/><circle id="path4157-35" class="st4" cx="-173.4" cy="283.6" r="4.2"/><circle id="path4157-62" class="st4" cx="-131.7" cy="339" r="4.2"/><ellipse id="path4157-9" transform="rotate(-89.334 -65.419 284.76) scale(.99997)" class="st4" cx="-65.4" cy="284.8" rx="14.7" ry="14.7"/></g></g></svg>
```

Comment:
This is actual logo of Kyber Network from pages - https://kyber.network/app/images/kyber-logo.svg (but its logo-full). I think logo-square can be easily extracted from this

Reply:
I have found the logo-square in svg, but did not match the logo-full. I did not find any information about the logo change, so there's a possibility this could be correct one:
https://worldvectorlogo.com/logo/knc